packages/preferences/src/node/preference-backend-module.ts

```diff
@@ -27,6 +28,13 @@ export const UserPreferenceServer = Symbol('UserPreferenceServer');
 export type UserPreferenceServer = PreferenceServer;
 
 export default new ContainerModule(bind => {
+
+    bind(PreferenceSchemaFactory).toSelf().inSingletonScope();
```

Comment:
https://github.com/inversify/InversifyJS/blob/master/wiki/factory_injection.md#injecting-a-factory

Look for `FileDialogFactory` for example.